py4DSTEM/io/filereaders/read_K2.py

```diff
@@ -124,7 +124,8 @@ def __init__(
             # this may be wrong for binned data... in which case the reader doesn't work anyway!
             Q_Nx = gtg.allTags[".SI Image Tags.Acquisition.Parameters.Detector.height"]
             Q_Ny = gtg.allTags[".SI Image Tags.Acquisition.Parameters.Detector.width"]
-        except:
+        # TODO check this is the correct error type
```

Comment:
TODO

Reply:
I can remove as `Exception` will handle what we want, but I think it would be better to add the expected exception/s in the future, but I'm  not familiar enough to check/test